starclass/BaseClassifier.py

```diff
@@ -489,3 +549,42 @@ def parse_labels(self, labels):
 			else:
 				fitlabels.append(lbl[0].value)
 		return np.array(fitlabels)
+
+#----------------------------------------------------------------------------------------------
+	def compute_ROC(self, y_test, y_prob, class_names):
+		"""
```

Comment:
Add empty lines between "sections" of docstring and explain what each parameter/result is.